addon-test-support/@ember/test-helpers/dom/blur.js

```diff
@@ -0,0 +1,37 @@
+import getElement from './-get-element';
+import fireEvent from './fire-event';
+import settled from '../settled';
+import isFocusable from './-is-focusable';
+
+/*
+  @method blur
+  @param {String|HTMLElement} selector
+  @return {RSVP.Promise}
+  @public
+*/
+export default function blur(selectorOrElement) {
```

Comment:
does it make sense that `blur()` takes a selector? should it default to `document.activeElement`?

Reply:
Seems reasonable...

Reply:
we should check if that works properly with headless browsers and unfocused tabs though